libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -728,6 +730,8 @@ void AP_MotorsMulticopter::output_motor_mask(float thrust, uint16_t mask, float
     const int16_t pwm_min = get_pwm_output_min();
     const int16_t pwm_range = get_pwm_output_max() - pwm_min;
 
+    _motor_mask_override = mask;
+
     for (uint8_t i = 0; i < AP_MOTORS_MAX_NUM_MOTORS; i++) {
         if (motor_enabled[i]) {
             if ((mask & (1U << i)) && armed() && get_interlock()) {
```

Comment:
With the new mask, I think we should zeroing motors that were not using. That should mean they spool up and down nicely rather. Although we might still have to do a better job of spool states in plane.

Something like:
```
    for (uint8_t i = 0; i < AP_MOTORS_MAX_NUM_MOTORS; i++) {
        if (motor_enabled[i] && (mask & (1U << i))
            if (armed() && get_interlock()) {
                /*
                apply rudder mixing differential thrust
                copter frame roll is plane frame yaw as this only
                apples to either tilted motors or tailsitters
                */
                float diff_thrust = get_roll_factor(i) * rudder_dt * 0.5f;
                set_actuator_with_slew(_actuator[i], thrust + diff_thrust);
            } else {
                // Zero throttle
                _actuator[i] = 0.0;
            }
            int16_t pwm_output = pwm_min + pwm_range * _actuator[i];
            rc_write(i, pwm_output);
        }
    }
```

Reply:
I wonder if we should also only allow override in the shutdown spool state, such that this will only work when there is no conflict with `output_to_motors`. It would make it very clear if were doing it wrong.